basic-caching/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "basic-caching",
+    "version": "1.0.0",
+    "description": "API proxy that demonstrates a basic caching policies",
```

Comment:
`demonstrates a basic` -> `demonstrates basic`

Reply:
Corrected in [e6a6bf4](https://github.com/GoogleCloudPlatform/apigee-samples/pull/67/commits/e623d016e9a063cf5f38696a7a7008b18e6a6bf4)